styles2.css

```diff
@@ -1,211 +1,288 @@
-  background: #ebebff;
-  border-radius: 8px;
-}
+  .info {
+    max-width: 1200px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: auto;
+  }
 
-#frame2 h1 {
-  width: 55px;
-  height: 16px;
-  font-family: "Poppins", sans-serif;
-  font-weight: 700;
-  font-size: 13px;
-  line-height: 16px;
-  color: #344563;
-}
+  .about-main {
+    background-image: url(./Image/Header\ bg2x.png);
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    min-height: 110vh;
+    position: relative;
+    top: 50px;
+    margin-bottom: 60px;
+  }
 
-#frame2 h2 {
-  width: 88px;
-  height: 16px;
-  font-family: "Poppins", sans-serif;
-  font-weight: 600;
-  font-size: 13px;
-  line-height: 16px;
-  color: #7a869a;
-}
+  .contain {
+    max-width: 1000px;
+    padding: 24px;
+    font-family: "Poppins", sans-serif;
+    position: absolute;
+    left: 10%;
+    top: 10%;
+  }
 
-#counter-Img {
-  background-image: url(images/Counter.png);
-  width: 8px;
-  height: 8px;
-  display: flex;
-}
+  .about-main .intro {
+    margin-top: 170px;
+    margin-left: 90px;
+    line-height: 56px;
+    font-size: 48px;
+  }
 
-#frame2 h4 {
-  color: #7a869a;
-  width: 29px;
-  height: 16px;
-  font-family: "Poppins", sans-serif;
-  font-weight: 600;
-  font-size: 13px;
-  line-height: 16px;
-}
+  .about p {
+    margin-left: 90px;
+    font-size: 18px;
+    line-height: 28px;
+  }
 
-.work-text p {
-  width: 295px;
-  height: 76px;
-  font-family: "Poppins", sans-serif;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #344563;
-}
+  .connect,
+  .social-icons {
+    margin-left: 90px;
+  }
 
-.Tags {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 0;
-  gap: 8px;
-  width: 214px;
-  height: 24px;
-}
+  .work-sections {
+    max-width: 1200px;
+    margin: auto;
+  }
 
-.button {
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 16px;
-  gap: 12px;
-  margin-top: 40px;
-  margin-bottom: 70px;
-  background: #fff;
-  border: 1px solid #dfe1e6;
-  border-radius: 16px;
-  color: #dfe1e6;
-}
+  .container {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
 
-.button1 {
-  background-color: white;
-  border: 2px solid #396df2;
-  font-family: "Poppins", sans-serif;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #396df2;
-  margin-bottom: 40px;
-}
+  .works-container {
+    padding: 30px;
+    display: flex;
+    justify-content: space-between;
+    max-width: 1000px;
+    margin: 50px auto;
+    transition: all 0.3s ease-in;
+    align-items: center;
+  }
 
-.topframe {
-  flex-direction: column;
-}
+  .works-container > * {
+    flex: 1;
+  }
 
-.frame-list ul li {
-  flex-direction: column;
-}
+  .works-container:hover {
+    transform: scale(0.9);
+  }
```

Comment:
- [ ] Ooo I really like that subtle animation 👏 but please make sure you're also following the Figma design as shown below:
    ![firefox_JmniUFjepy](https://user-images.githubusercontent.com/104968236/230642990-f8ebf907-38c5-41fb-aaf8-3b193de285f4.png)
    - As you can see when the mouse hovers over the project card there's a border with a `#A7AEFF` color and a subtle box shadow so please make sure you add that.✔